src/js/components/pages/scenegraph.jsx

```diff
@@ -356,14 +376,14 @@ var SceneGraph = React.createClass({
                 </div>
             );
         }
-        try{
+        try {
             var isAdmin = ConnectionCache.getGroupID() === "0";
 
-            var regenerateButton = isAdmin?(<Button className="btn btn-info"
-                                                   onClick={this.regenerateSceneGraphHandler}>
+            var regenerateButton = isAdmin ? (<Button className="btn btn-info"
```

Comment:
Because this is declared within a try, catch scope I don't see how the reference is available inside the return?